scripts/verify-turnin

```diff
@@ -32,105 +32,138 @@
 #                                                                        #
 ##########################################################################
 
-import sys,os
-import fnmatch, re, getpass
+import fnmatch
+import getpass
+import os # sometime maybe replace os with subproccess for additional control
+import subprocess
+import re
+import sys
 
-ver = "1.0"
+
+ver = None;
 
 def usage():
-    print (" verify-turnin v" + ver)
-    print " Usage: "
-    print " verify-turnin assignment@course "
-    print " \tVerify the turnin has been successful "
-    print " verify-turnin -V "
-    print " \tShow Copyright and Version information "
+    print(ver)
+    print(" Usage: ")
+    print(" verify-turnin assignment@course ")
+    print(" \tVerify the turnin has been successful ")
+    print(" verify-turnin -V ")
+    print(" \tShow Copyright and Version information ")
+
 
 def versionstuff():
-    print (" verify-turnin v" + ver)
-    print " Copyright: "
-    print " Copyright (c) 2014 \tAntonios A. Chariton \t<daknob.mac@gmail.com> "
-    print " License: "
-    print " The current file is under the MIT License. "
-    print " You can obtain a copy of the license here: <http://opensource.org/licenses/MIT> "
-    print " Development: "
-    print " Source Code available on GitHub: <https://github.com/zakkak/turnin> "
+    print(ver)
+    #print(" Copyright: ")
+    #print(" Copyright (c) 2014 \tAntonios   A. Chariton    \t<daknob.mac@gmail.com> ")
+    #print(" Copyright (c) 2022 \tAlexandros K. Antonakakis \t<antonakakis1@gmail.com> ")
+    print(" License: ")
+    print(" The current file is under the MIT License. ")
```

Comment:
```suggestion
    print(" The current file is licensed under the MIT License. ")
```

Reply:
MIT doesn't require this print. So I suggest either keeping the whole thing (i.e. copyright notices and link) or completely removing the licensing info from `-V`